### PR TITLE
Updating Collecting log files section

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for issue https://github.com/eclipse/codewind/issues/2851.
Editing the Troubleshooting section, "Collecting log files and environment data."

### CODE REVIEW

**Comment by sishida** [comment]
@eharris369, @tetchel: Hi, team, I've created this PR to talk about collecting diagnostics from the IDEs.

**Quick question:** Now that users can access this info from the IDEs, do we still want to mention the `cwctl` command? Do **Codewind: Capture Diagnostics** and **Collect Diagnostics** replace `cwctl diagnostics collect <command options> <arguments...>`? Thanks!

**Comment by tetchel** [comment]
We can leave the `cwctl` bit, it still applies.

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -842,26 +842,29 @@ To assist with problem determination, raise the default Codewind server log leve
...
 Action/Topic: Collecting log files and environment data
 Issue type: info
-Issue link: https://github.com/eclipse/codewind/issues/2766 https://github.com/eclipse/codewind/issues/2930
-Info added in 0.12.0, updated in 0.13.0.
+Issue links: https://github.com/eclipse/codewind/issues/2766 https://github.com/eclipse/codewind/issues/2930 https://github.com/eclipse/codewind/issues/2851
+Info added in 0.12.0 and updated in 0.13.0.
 -->
 
-You can capture diagnostics from your installation by using the `cwctl diagnostics collect` CLI command to collect all available log files and environment information. You can find the `cwctl` CLI in your HOME directory under the `~/.codewind/<version>` path.
+In Eclipse and VS Code, capture diagnostics from your installation by using the `cwctl diagnostics collect` CLI command to collect all available log files and environment information. Find the `cwctl` CLI in your HOME directory in the `~/.codewind/<version>` path.
```
Comment (eharris369) [inline docs/_documentations/troubleshooting.md:849]: Move this below the instructions to collect diagnostics using the IDEs. Using the IDEs is probably the easier way of doing it for users, plus this keeps the instructions for using cwctl together with the command format below.

**Comment by sishida** [comment]
@eharris369, @tetchel Thank you for your feedback! I have incorporated the suggested changes and made some other changes, as well. Please see the latest commit. Thank you!

**Pull request merged**